src/test/java/com/o19s/es/ltr/query/StoredLtrQueryIT.java

```diff
@@ -56,6 +59,8 @@ public void testFullUsecase() throws Exception {
                 QueryBuilders.matchQuery("field1", "{{query}}").toString()));
         addElement(new StoredFeature("feature2", Collections.singletonList("query"), "mustache",
                 QueryBuilders.matchQuery("field2", "{{query}}").toString()));
+        addElement(new StoredFeature("feature3", Collections.singletonList("query"), "derived_expression",
+                "(feature1 + feature2) * 5"));
```

Comment:
maybe you can test:
- set feature3 weight to 10
- use expression `signum(feature1 - feature2)` => should return -1 if feature2 if non zero, 1 if feature1 is non zero
Then you could assert that the score is > 10 if negativeScore == false, < -10 is negativeScore is true